.envrc

```diff
@@ -120,7 +120,7 @@ export AIR_CONFIG_DEBUG="-c .airDebug.conf"
 export AIR_CONFIG_DEBUG_WAIT="-c .airDebugWait.conf"
 
 # Faktory Config
-export FAKTORY_PROCESS_JOBS=false
+export FAKTORY_PROCESS_JOBS=true
```

Comment:
Probably don't want to commit this new default -- perhaps this was meant to be added to your `.envrc.local`?